src/app/charity/charity.component.html

```diff
@@ -1,39 +1,46 @@
 <main class="b-container">
   <div class="c-header">
-    <h1 *ngIf="campaigns[0]" class="b-rh-1 b-bold">{{ campaigns[0].charity.name }} Campaigns</h1>
+    @if (campaigns[0]) {
+      <h1 class="b-rh-1 b-bold">{{ campaigns[0].charity.name }} Campaigns</h1>
+    }
   </div>
 
   <div>
-    <div *ngIf="campaigns.length === 0">
-      <p class="error" aria-live="polite">
-        None yet! <a tabindex="0" routerLink="/">Find campaigns here</a>.
-      </p>
-    </div>
+    @if (campaigns.length === 0) {
+      <div>
+        <p class="error" aria-live="polite">
+          None yet! <a tabindex="0" routerLink="/">Find campaigns here</a>.
+        </p>
+      </div>
+    }
 
-    <biggive-grid
+    @if (campaigns) {
+      <biggive-grid
         slot="campaign-grid"
         column-count="3"
-        *ngIf="campaigns"
-      >
-        <div *ngFor="let campaign of campaigns">
-          <biggive-campaign-card
-            id="campaign-{{campaign.id}}"
-            [banner]="campaign.imageUri | optimisedImage:830 | async"
-            [campaignType]="campaign.isMatched ? 'Match Funded' : null"
-            [campaignTitle]="campaign.title"
-            [donateButtonUrl]="'/donate/' + campaign.id"
-            [moreInfoButtonUrl]="'/campaign/' + campaign.id"
-            [organisationName]="campaign.charity.name"
-            primary-figure-label="Match Funds Remaining"
-            [primaryFigureAmount]="campaign.matchFundsRemaining | currency: campaign.currencyCode : 'symbol' : currencyPipeDigitsInfo"
-            [progressBarCounter]="getPercentageRaised(campaign)"
-            secondary-figure-label="Total Funds Raised"
-            [secondaryFigureAmount]="campaign.amountRaised | currency: campaign.currencyCode : 'symbol' : currencyPipeDigitsInfo"
-            [isFutureCampaign]="isInFuture(campaign)"
-            [isPastCampaign]="isInPast(campaign)"
-            [datetime]="getRelevantDateAsStr(campaign)">
-          </biggive-campaign-card>
-        </div>
+        >
+        @for (campaign of campaigns; track campaign) {
```

Comment:
```suggestion
        @for (campaign of campaigns; track campaign.id) {
```

I guess this would be more performant? Think same should work for all 3 places on this PR where we `track campaign`.

Reply:
@NoelLH . Yes. I read https://angular.io/api/core/for#description . On this page the list of campaigns is static so I think the simplest option is `track $index`. On the explore and campaign changes the list of campaigns can change so I went with track `campaign.id` as you suggested. Commit 4887f533

Reply:
And applied the same thinking to other for loops in 44a4bcd8